flake.nix

```diff
@@ -0,0 +1,59 @@
+{
+  description = "A tree-sitter grammar for Nickel";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+        version = "0.2.0";
+        tree-sitter-nickel = pkgs.callPackage (nixpkgs + "/pkgs/development/tools/parsing/tree-sitter/grammar.nix") { }
+          {
+            language = "nickel";
+            src = ./.;
+            inherit version;
+          };
+      in
+      {
+        packages.default = tree-sitter-nickel;
+
+        devShells.default = pkgs.mkShell {
+          nativeBuildInputs = with pkgs; [
+            nodejs_latest
+            tree-sitter
+            clang
+          ];
+          profile = ''
+            export npm_config_build_from_source=true
+          '';
+        };
+
+        checks.default =
```

Comment:
Should we use `nix flake check` instead of `npm` for the CI now? (it's an open question, and should be a different PR anyway)

Reply:
Yes! That is probably a good idea!